api/scripts/healthcheck.py

```diff
@@ -1,25 +1,31 @@
-import logging
-import sys
-
-import requests
-
-HEALTH_LIVENESS_URL = "http://localhost:8000/health/liveness"
-
-
-logger = logging.getLogger(__name__)
+import argparse
+import socket
+
+parser = argparse.ArgumentParser(
+    description="Check if the API is running and accessible.",
```

Comment:
```suggestion
    description="Check if the API is able to accept local TCP connections.",
```

"Accessible" is not necessarily accurate if the request queue is full or something else (like DB being down, network etc) prevents it from working.